examples/dt/uni07eta/networker.md

```diff
@@ -0,0 +1,39 @@
+# Configuring and deploying Networker nodes
+
+## Assumptions
+
+- The [control plane](control-plane.md) has been created and successfully
+  deployed.
+
+## Steps
+
+```bash
+# 1. Switch to openstack namespace
+
+    $ oc project openstack
+
+# 2. Change the working directory to uni-eta
+
+    $ pushd architecture/examples/dt/uni07eta
+
+# 3. Modify [networker/values.yaml](networker/values.yaml) file to suit your
+#    environment.
+
+    $ pushd networker
+    $ vi values.yaml
+
+# 4. Generate the networker data plane deployment plan.
+
+    $ kustomize build --load-restrictor LoadRestrictionsNone > \
```

Comment:
Can we build the `edpm-networker.yaml` without `--load-restrictor LoadRestrictionsNone`? 

Right now it has no effect on the default samples.

```
[fultonj@runcible uni07eta{uni-etha}]$ pwd
/home/fultonj/review/architecture/examples/dt/uni07eta
[fultonj@runcible uni07eta{uni-etha}]$ kustomize build > one.yaml
[fultonj@runcible uni07eta{uni-etha}]$  kustomize build --load-restrictor LoadRestrictionsNone > two.yaml
[fultonj@runcible uni07eta{uni-etha}]$ md5sum one.yaml two.yaml 
66ae6992f58646c683d36b7a3dc51b42  one.yaml
66ae6992f58646c683d36b7a3dc51b42  two.yaml
[fultonj@runcible uni07eta{uni-etha}]$
```